static/js/admin/src/action/ChangeLocationAction.js

```diff
@@ -5,15 +5,12 @@ define('ChangeLocationAction', [ 'jquery'], function($) {
 	
 	ChangeLocationAction.prototype.execute = function() {
 		var url;
-		if (this.context.application == "access") {
+		if (this.context.application !== undefined) {
```

Comment:
where does this.context.application get set? and does this change mean that as long as some value is stored in that var we want to set the url to accessBaseUrl?